source/agora/consensus/Ledger.d

```diff
@@ -1295,22 +1326,24 @@ public class Ledger
     }
 }
 
-private Height lastPaymentBlock(in Height height, uint payout_period) @safe @nogc nothrow pure
+// This is the last block height that has had fees and rewards paid before the current block
+private Height lastPaidHeight(in Height height, uint payout_period) @safe @nogc nothrow pure
 {
-    return Height(height <= payout_period ? 1 : (height - 1) - ((height - 1) % payout_period));
+    // We return 1 before the first payout is made as we use this for block catchup and Genesis is never fetched
```

Comment:
We don't use it for block catchup AFAICS, only for signature catchup ?

Reply:
Correct I will update.